src/components/Episodes/data.js

```diff
@@ -1,22 +1,22 @@
 export default [
 	{
 		number: 1,
-		image: {
+		thumbnail: {
```

Comment:
didn't we already do this? is this a duplication of work from the other branch?

Reply:
err we did this ages days ago 🤔